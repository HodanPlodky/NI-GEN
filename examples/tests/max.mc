int max(int a, int b) {
    if (a < b)
        return b;
    else 
        return a;
}

int main() {
    int x = 7;
    int y = 18;
    return max(max(10, y), max(2, x));
}
