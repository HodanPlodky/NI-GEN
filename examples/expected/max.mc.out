input file : examples/tests/max.mc
return 18
